spec/feature/authenticated_admin_spec.rb

```diff
@@ -266,81 +266,81 @@
         allow_any_instance_of(ApplicationController).to receive(:current_user).
         and_return(admin)
-        create_user_orders_with_items
-        visit admin_orders_path
-        within(".orders-list") do
-          click_link_or_button "Order 00001"
+        create_user_reservations_with_items
+        visit admin_reservations_path
+        within(".reservations-list") do
+          click_link_or_button "Reservation 00001"
         end
-        within("#order-status") do
+        within("#reservation-status") do
           click_link_or_button "change to paid"
         end
-        within("#order-status") do
+        within("#reservation-status") do
           expect(page).to have_content("Status: paid")
         end
       end
 
-      xit "'mark as completed' individual orders which are currently 'paid'" do
+      xit "'mark as completed' individual reservations which are currently 'paid'" do
         allow_any_instance_of(ApplicationController).to receive(:current_user).
                                                               and_return(admin)
-        create_user_orders_with_items
-        visit admin_orders_path
-        within(".orders-list") do
-          click_link_or_button "Order 00001"
+        create_user_reservations_with_items
+        visit admin_reservations_path
+        within(".reservations-list") do
+          click_link_or_button "Reservation 00001"
         end
-        within("#order-status") do
+        within("#reservation-status") do
           click_link_or_button "change to complete"
         end
-        within("#order-status") do
+        within("#reservation-status") do
           expect(page).to have_content("Status: completed")
         end
       end
 
-      xit "'cancel' individual orders which are currently 'ordered' or 'paid'" do
+      xit "'cancel' individual reservations which are currently 'ordered' or 'paid'" do
         allow_any_instance_of(ApplicationController).to receive(:current_user).
                                                               and_return(admin)
-        create_user_orders_with_items
-        visit admin_orders_path
-        within(".orders-list") do
-          click_link_or_button "Order 00001"
+        create_user_reservations_with_items
+        visit admin_reservations_path
+        within(".reservations-list") do
+          click_link_or_button "Reservation 00001"
         end
-        within("#order-status") do
+        within("#reservation-status") do
           click_link_or_button "change to cancel"
         end
-        within("#order-status") do
+        within("#reservation-status") do
           expect(page).to have_content("Status: cancelled")
         end
       end
     end
   end
 
-  def create_user_orders_with_items
+  def create_user_reservations_with_items
     breakfast = Category.create(name: "Breakfast")
 
     item1 = Item.new(title: "Bacon and Eggs",
```

Comment:
Agreed, I will go in and modify these tests as we build out the new functionality